lms/djangoapps/certificates/tests/test_tasks.py

```diff
@@ -32,4 +39,30 @@ def test_cert_task_missing_args(self, missing_arg):
 
         with patch('lms.djangoapps.certificates.tasks.User.objects.get'):
             with self.assertRaisesRegexp(KeyError, missing_arg):
-                generate_certificate(**kwargs)
+                generate_certificate.apply_async(kwargs=kwargs).get()
+
+    @patch('lms.djangoapps.certificates.tasks.generate_user_certificates')
+    @patch('lms.djangoapps.verify_student.models.SoftwareSecurePhotoVerification.user_status')
+    def test_cert_task_retry(self, user_status_mock, generate_user_certs_mock):
```

Comment:
Nitty: you can remove the "cert_task" from the name of each test method.  The cert context is already assumed from the name of the test class (`GenerateUserCertificateTest`), while the task context is already assumed from  the name of the module (`test_tasks.py`).